docs/usage/client-cli.md

```diff
@@ -20,19 +20,25 @@ Information about the Walrus system is available through the `walrus info` comma
 $ walrus info
 
 Walrus system information
+Current epoch: 0
 
 Storage nodes
 Number of nodes: 10
 Number of shards: 270
 
 Blob size
 Maximum blob size: 957 MiB (1,003,471,920 B)
+Storage unit: 1.00 KiB
 
 Approximate storage prices per epoch
-Price per encoded storage unit: 50 MIST/KiB
+Price per encoded storage unit: 50 MIST
 Price to store metadata: 850 MIST
```

Comment:
I just noticed that the maximum blob size is also not correct anymore after switching to 1000 shards. It might be easiest to just replace the full example output with the output of running the `info` command against a local deployment with 1000 shards (this will also change the blob sizes for which the cost is provided).